diff -urN libgtop-2.0.1.orig/acinclude.m4 libgtop-2.0.1/acinclude.m4
--- libgtop-2.0.1.orig/acinclude.m4	Mon Jan 13 22:23:10 2003
+++ libgtop-2.0.1/acinclude.m4	Mon Jan 13 22:23:33 2003
@@ -517,10 +517,10 @@
   # this should go away soon
   need_gnome_support=yes
 
-  save_LIBOBJS="$LIBOBJS"
-  LIBOBJS=
-
-  AC_CHECK_FUNCS(getopt_long,,LIBOBJS="$LIBOBJS getopt.o getopt1.o")
+  AC_CHECK_FUNCS(getopt_long,,[
+  	AC_LIBOBJ([getopt])
+	AC_LIBOBJ([getopt1])
+  ])
 
   # for `scandir'
   AC_HEADER_DIRENT
@@ -545,7 +545,7 @@
   AC_REPLACE_FUNCS(memmove mkstemp scandir strcasecmp strerror strndup strnlen)
   AC_REPLACE_FUNCS(strtok_r strtod strtol strtoul vasprintf vsnprintf)
 
-  AC_CHECK_FUNCS(realpath,,LIBOBJS="$LIBOBJS canonicalize.o")
+  AC_CHECK_FUNCS(realpath,,[AC_LIBOBJ([canonicalize])])
 
   # to include `error.c' error.c has some HAVE_* checks
   AC_CHECK_FUNCS(vprintf doprnt strerror_r)
@@ -555,9 +555,8 @@
               am_cv_lib_error_at_line=yes,
               am_cv_lib_error_at_line=no)])
   if test $am_cv_lib_error_at_line = no; then
-   LIBOBJS="$LIBOBJS error.o"
+   AC_LIBOBJ([error])
   fi
-  AC_SUBST(LIBOBJS)dnl
   
   # This is required if we declare setreuid () and setregid ().
   AC_TYPE_UID_T
@@ -574,11 +573,11 @@
 
   # Turn our LIBOBJS into libtool objects.  This is gross, but it
   # requires changes to autoconf before it goes away.
-  LTLIBOBJS=`echo "$LIBOBJS" | sed 's/\.o/.lo/g'`
+  LTLIBOBJS=`echo "$LIB@&t@OBJS" |                                                                         
+   sed 's,\.[[^.]]* ,.lo ,g;s,\.[[^.]]*$,.lo,'`                                                  
   AC_SUBST(need_gnome_support)
   AC_SUBST(LTLIBOBJS)
 
-  LIBOBJS="$save_LIBOBJS"
   AM_CONDITIONAL(BUILD_GNOME_SUPPORT, test "$need_gnome_support" = yes)
 ])
 
diff -urN libgtop-2.0.1.orig/configure.in libgtop-2.0.1/configure.in
--- libgtop-2.0.1.orig/configure.in	Mon Jan 13 22:23:10 2003
+++ libgtop-2.0.1/configure.in	Mon Jan 13 22:23:33 2003
@@ -51,6 +51,7 @@
 AC_ISC_POSIX
 AC_PROG_CC
 AC_STDC_HEADERS
+AM_PROG_AS
 dnl AC_ARG_PROGRAM
 
 dnl We need this at some places.
