diff -aurN libgtop-2.14.2.orig/src/inodedb/Makefile.am libgtop-2.14.2/src/inodedb/Makefile.am
--- libgtop-2.14.2.orig/src/inodedb/Makefile.am	2003-10-20 22:47:58.000000000 +0200
+++ libgtop-2.14.2/src/inodedb/Makefile.am	2006-08-13 19:07:36.922763444 +0200
@@ -2,8 +2,8 @@
 
 INCLUDES		= @INCLUDES@
 
-mkinodedb2_LDADD		= $(top_builddir)/lib/libgtop-2.0.la \
-			  $(top_builddir)/sysdeps/common/libgtop_common-2.0.la \
+mkinodedb2_LDADD		= $(top_builddir)/sysdeps/common/libgtop_common-2.0.la \
+			  $(top_builddir)/lib/libgtop-2.0.la \
 			  $(top_builddir)/sysdeps/@sysdeps_dir@/libgtop_sysdeps-2.0.la
 
 file_by_inode2_LDADD	= $(top_builddir)/lib/libgtop-2.0.la \
