diff -Nur libgtop-1.90.2/acinclude.m4 libgtop-1.90.2.new/acinclude.m4
--- libgtop-1.90.2/acinclude.m4	Sun Feb 10 15:30:14 2002
+++ libgtop-1.90.2.new/acinclude.m4	Sun Apr 28 20:14:59 2002
@@ -517,10 +517,10 @@
   # this should go away soon
   need_gnome_support=yes
 
-  save_LIBOBJS="$LIBOBJS"
-  LIBOBJS=
+#  save_LIBOBJS="$LIBOBJS"
+#  LIBOBJS=
 
-  AC_CHECK_FUNCS(getopt_long,,LIBOBJS="$LIBOBJS getopt.o getopt1.o")
+  AC_CHECK_FUNCS(getopt_long,,AC_LIBOBJ([getopt getopt1]))
 
   # for `scandir'
   AC_HEADER_DIRENT
@@ -545,7 +545,7 @@
   AC_REPLACE_FUNCS(memmove mkstemp scandir strcasecmp strerror strndup strnlen)
   AC_REPLACE_FUNCS(strtok_r strtod strtol strtoul vasprintf vsnprintf)
 
-  AC_CHECK_FUNCS(realpath,,LIBOBJS="$LIBOBJS canonicalize.o")
+  AC_CHECK_FUNCS(realpath,,AC_LIBOBJ([canonicalize]))
 
   # to include `error.c' error.c has some HAVE_* checks
   AC_CHECK_FUNCS(vprintf doprnt strerror_r)
@@ -555,9 +555,9 @@
               am_cv_lib_error_at_line=yes,
               am_cv_lib_error_at_line=no)])
   if test $am_cv_lib_error_at_line = no; then
-   LIBOBJS="$LIBOBJS error.o"
+   AC_LIBOBJ([error])
   fi
-  AC_SUBST(LIBOBJS)dnl
+#  AC_SUBST(LIBOBJS)dnl
   
   # This is required if we declare setreuid () and setregid ().
   AC_TYPE_UID_T
@@ -574,11 +574,13 @@
 
   # Turn our LIBOBJS into libtool objects.  This is gross, but it
   # requires changes to autoconf before it goes away.
-  LTLIBOBJS=`echo "$LIBOBJS" | sed 's/\.o/.lo/g'`
+  LIB@&t@OBJS=`echo "$LIB@&t@OBJS" | sed 's,\.[[^.]]* ,$U&,g;s,\.[[^.]]*$,$U&,'`
+  LTLIBOBJS=`echo "$LIB@&t@OBJS" | sed 's,\.[[^.]]* ,.lo ,g;s,\.[[^.]]*$,.lo,'`
+				       
   AC_SUBST(need_gnome_support)
   AC_SUBST(LTLIBOBJS)
 
-  LIBOBJS="$save_LIBOBJS"
+#  LIBOBJS="$save_LIBOBJS"
   AM_CONDITIONAL(BUILD_GNOME_SUPPORT, test "$need_gnome_support" = yes)
 ])
 
@@ -638,7 +640,7 @@
 	  if test $linux_table = yes ; then
 	    AC_DEFINE(HAVE_LINUX_TABLE)
 	  fi
-	  AM_CONDITIONAL(LINUX_TABLE, test $linux_table = yes)
+#	  AM_CONDITIONAL(LINUX_TABLE, test $linux_table = yes)
 	  ;;
 	esac
 ])
@@ -873,4 +875,4 @@
 	AM_CONDITIONAL(NEED_LIBGTOP, test x$libgtop_need_server = xyes)
 ])
 
-  
\ No newline at end of file
+  
